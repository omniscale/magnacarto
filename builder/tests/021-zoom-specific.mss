// only zoom=15 has a line-width

#roads[zoom>=14] {
  line-color: white;
  [type='primary'] {
    // this should create a rule for level 15 with line-width: 5
    line-color: red;
  }
}

#roads[zoom=15] {
  line-width: 5;
}


#roads[zoom>=14] {
  line-cap: round;
}

#roads {
  line-join: bevel;
}