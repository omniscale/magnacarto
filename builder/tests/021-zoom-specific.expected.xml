<Map srs="epsg:3857">
  <Parameters></Parameters>
  <Style name="roads" filter-mode="first">
    <Rule>
      <!--Zoom{=15}-->
      <MaxScaleDenominator>25000</MaxScaleDenominator>
      <MinScaleDenominator>12500</MinScaleDenominator>
      <Filter>([type] = &#39;primary&#39;)</Filter>
      <LineSymbolizer stroke="#ff0000" stroke-linecap="round" stroke-linejoin="bevel" stroke-width="5"></LineSymbolizer>
    </Rule>
    <Rule>
      <!--Zoom{>=14}-->
      <MaxScaleDenominator>50000</MaxScaleDenominator>
      <Filter>([type] = &#39;primary&#39;)</Filter>
    </Rule>
    <Rule>
      <!--Zoom{=15}-->
      <MaxScaleDenominator>25000</MaxScaleDenominator>
      <MinScaleDenominator>12500</MinScaleDenominator>
      <LineSymbolizer stroke="#ffffff" stroke-linecap="round" stroke-linejoin="bevel" stroke-width="5"></LineSymbolizer>
    </Rule>
    <Rule>
      <!--Zoom{>=14}-->
      <MaxScaleDenominator>50000</MaxScaleDenominator>
    </Rule>
    <Rule></Rule>
  </Style>
  <Layer name="roads" srs="" status="off">
    <StyleName>roads</StyleName>
  </Layer>
</Map>